src/platform/silabs/BLEManagerImpl.h

```diff
@@ -25,6 +25,12 @@
 #pragma once
 #if CHIP_DEVICE_CONFIG_ENABLE_CHIPOBLE
 
+#ifdef RSI_BLE_ENABLE
+#define MIN_CONNECTION_INT 45 // 45 msec
+#define MAX_CONNECTION_INT 45 // 45 msec
+#define SLAVE_LATENCY 0
+#define TIMEOUT 400
```

Comment:
What is timeout and slave latency measured in? MS?

`TIMEOUT` is a very generic name, not obviously BLE. C++-namespace it for ble or name it `BLE_TIMEOUT_MS` or similar.

Reply:
Changed in latest commit